study/cyber-tutor/index.html

```diff
@@ -0,0 +1,91 @@
+<html>
+    <head>
+        <title>Cyber Tutor</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link rel="stylesheet" href="../../css/bootstrap4.5.0.min.css">
+        <link rel="stylesheet" href="../codemirror/codemirror.css">
+        <link rel="stylesheet" href="css/style.css">
+
+        <script src="https://kit.fontawesome.com/b3cde37b04.js" crossorigin="anonymous"></script>
+    </head>
+
+    <body>
+        <nav class="navbar navbar-expand-md bg-secondary navbar-dark mb-3">
+            <a class="navbar-brand" href="../../">TeCanal Kids Corner</a>
+
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="collapsibleNavbar">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="../../play">
+                            <i class="fa fa-gamepad"></i>
+                            Play
+                        </a>
+                    </li>
+
+                    <li class="nav-item active">
```

Comment:
This is missing the <a> element, which makes it a non-functional navigation menu item. It should look something like below, so check all of your pages.
```html
<a class="nav-link" href="../../study">
     <i class="fa fa-paperclip"></i>
     Study
</a>
```